packages/blade/src/utils/getColorScheme/getColorScheme.native.ts

```diff
@@ -1,14 +1,14 @@
 import { Appearance } from 'react-native';
-import { ColorSchemeNames } from '../../tokens/theme';
+import { ColorSchemeNames, ColorSchemeNamesInput } from '../../tokens/theme';
 
-const getColorScheme = (colorScheme: ColorSchemeNames): Exclude<ColorSchemeNames, 'system'> => {
+const getColorScheme = (colorScheme: ColorSchemeNamesInput): ColorSchemeNames => {
   // @TODO: convert this to hook as Appearance API also adds an eventListener which subsribes to the colorscheme changes on the device
   if (colorScheme === 'light' || colorScheme === 'dark') {
     return colorScheme;
   }
 
   if (colorScheme === 'system') {
-    return Appearance.getColorScheme() || 'light';
+    return Appearance.getColorScheme() ?? 'light';
```

Comment:
is this change supposed to be here?

Reply:
didn't get you?

Reply:
oh I mean it doesn't look like related to release so is this an intentional change?

Reply:
If you mean if this change is contextual to this PR then yes since I updated few packages and ts-eslint started yelling at me and also I've added this to github check now else it'll fail the moment I merge this to master